Workbooks/ADXvsLA.json

```diff
@@ -0,0 +1,1428 @@
+{
+  "version": "Notebook/1.0",
+  "items": [
+    {
+      "type": 1,
+      "content": {
+        "json": "## ADX vs LA\n---\n\n"
+      },
+      "name": "text - 2"
+    },
+    {
+      "type": 9,
+      "content": {
+        "version": "KqlParameterItem/1.0",
+        "parameters": [
+          {
+            "id": "dfc22c7d-754c-417f-b2d2-9f70c0b1d439",
+            "version": "KqlParameterItem/1.0",
+            "name": "Subscription",
+            "label": "Subscriptions",
+            "type": 6,
+            "description": "All subscriptions with ADX Cluster",
+            "isRequired": true,
+            "query": "Resources\r\n| where type =~ 'Microsoft.Kusto/clusters'\r\n| summarize Count = count() by subscriptionId\r\n| order by Count desc\r\n| extend Rank = row_number()\r\n| project value = subscriptionId, label = subscriptionId, selected = Rank == 1",
+            "typeSettings": {
+              "additionalResourceOptions": [],
+              "showDefault": false
+            },
+            "timeContext": {
+              "durationMs": 86400000
+            },
+            "queryType": 1,
+            "resourceType": "microsoft.resourcegraph/resources"
+          },
+          {
+            "id": "bb362f52-6bde-4b4d-bae3-1765017c0106",
+            "version": "KqlParameterItem/1.0",
+            "name": "Resources",
+            "label": "Cluster",
+            "type": 5,
+            "description": "The ADX cluster",
+            "isRequired": true,
+            "query": "resources\r\n| where type =~ 'microsoft.kusto/clusters'",
+            "crossComponentResources": [
+              "{Subscription}"
+            ],
+            "typeSettings": {
+              "additionalResourceOptions": [],
+              "showDefault": false
+            },
+            "timeContext": {
+              "durationMs": 86400000
+            },
+            "queryType": 1,
+            "resourceType": "microsoft.resourcegraph/resources",
+            "value": "/subscriptions/1c61ccbf-70b3-45a3-a1fb-848ce46d70a6/resourceGroups/Admin/providers/Microsoft.Kusto/clusters/chiadxcluster"
```

Comment:
Subscription and Resource details should not be mentioned. Please update.
Validations are failing because of this.

Reply:
resolved